enhancements/observability-ui/observability-ui-operator.md

```diff
@@ -0,0 +1,265 @@
+---
+title: observability-ui-operator
+authors:
+  - "@jgbernalp"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@kyoto"
+  - "@zhuje"
+  - "@alanconway"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@stleerh"
+  - "@eparis"
+api-approvers:
+  - "@spadgett"
+creation-date: 2023-09-18
+last-updated: 2023-10-11
+tracking-link:
+  - https://issues.redhat.com/browse/OU-204
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+title: observability-ui-operator
+
+# Observability UI Operator
+
+The Observability UI Operator aims to manage dynamic console plugins for observability signals inside the OpenShift console, ensuring a consistent user experience and efficient management of UI plugins.
+
+## Summary
+
+This proposal introduces the Observability UI Operator, a tool designed to manage UI plugins related to observability signals within the OpenShift console. By centralizing the management of such plugins, we can offer a unified observability experience in the console and accommodate new use cases, all while decoupling UI responsibilities from other operators.
+
+## Motivation
+
+### Why:
+
+The current state of observability signals in the OpenShift console has each operator responsible for its own console plugin. This sometimes results in operators deploying plugins outside their primary scope. As the requirements for the console's UI grow, there's a clear need for a centralized system that can manage diverse UI components spanning across various signals to offer a unified observability experience in the console.
+
+### User Stories
+
+- As an OpenShift user, I want an operator from the Red Hat catalog that can deploy various observability UI components so that all signals supported by the cluster are easily accessible and can be used for troubleshooting.
+
+- As an OpenShift administrator, I want a centralized operator for observability UI components so that I can streamline console requirements and integrate diverse signals effectively.
+
+- As an OpenShift user, I want to customize observability dashboards with various signals so that I can quickly identify and resolve issues.
```

Comment:
Will this replace monitoring ?

Reply:
This will add new customizable dashboards, as the dashboards should not be specific to a single datasource. Totally agree that dashboards per team create disparity. Hence the goal is to add new dashboards that can consume multiple datasources and have richer charts so teams are not constrained in visualization. This is described in the design section: Dashboards console plugin.

Reply:
So it definitly makes sense for monitoring to be part of it and take advantage of the improvments ?
See https://github.com/openshift/enhancements/pull/1494#discussion_r1375881847

Reply:
Monitoring currently is more than only dashboards as it includes alerting rules and service monitors. So the part that is not coupled (dashboards) and offers other teams more advantages is the part we are extracting.